### ISSUE TITLE
Using TAA plug-in with ETS2/ATS Local radio cause the TAA plug-in to crash.

### ISSUE THREAD

**Issue by Ticreut29** [Koenvh1/ets2-local-radio#209]
Hi, unfortunately I an conflict with ETS2/ATS Local radio and TAA plug-in each other, it cause critical error 50801, -2005270527 and I need to close the game. There's a way to remove overlay to try if this fix the issue?

Link to the faulty plug-in: https://forum.scssoft.com/viewtopic.php?t=323315

**Comment by Koenvh1** [comment]
Does this also happen with the overlay turned off? (in `settings.json` set overlay to `false`)

**Comment by Ticreut29** [comment]
Yes the overlay is causing the issue, disable the overlay fix the issue and I can now use the TAA solution with the ETS/ATS Local Radio.
The issue happen when Alt + Tab when the overlay is enable.
